packages/react-core/src/components/Select/Select.tsx

```diff
@@ -428,6 +429,13 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
                   (valueToCheck as SelectOptionObject).toString &&
                   (valueToCheck as SelectOptionObject).compareTo;
 
+                // View more option should be returned as not a match
+                if (loadingVariant !== undefined && loadingVariant !== 'spinner') {
+                  if (loadingVariant?.text === valueToCheck && loadingVariant.text === valueToCheck) {
```

Comment:
Isn't this checking the same thing twice?

Reply:
seems like both both ifs could be replaced with a single `if (loadingVariant?.text === valueToCheck)` if it was either thing from the first condition, `loadingVariant?.text` would be `undefined` which could never be `valueToCheck` as we previously check that on 423.